src/promptflow/promptflow/_constants.py

```diff
@@ -38,3 +38,12 @@ class AvailableIDE:
 
 USER_AGENT = "USER_AGENT"
 PF_USER_AGENT = "PF_USER_AGENT"
+
```

Comment:
merge main once to make this available in your branch:

![image](https://github.com/microsoft/promptflow/assets/38847871/1b815e93-6fc7-4101-a374-8f7ad38d33cd)